trillian/integration/hammer_test.go

```diff
@@ -26,8 +26,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/golang/protobuf/ptypes"
-	"github.com/golang/protobuf/ptypes/timestamp"
+	timestamp "google.golang.org/protobuf/types/known/timestamppb"
```

Comment:
There are only a few references to this package below, I would leave the default name.